src/components/home/MapItem.vue

```diff
@@ -448,10 +448,10 @@ export default {
       let ratio = x / y
 
       if (ratio < this.threshold) { // portrait mode
-        map.style.transform = `scale(${y / 50})` + `rotate(90deg)`
+        map.style.transform = `scale(${y / 58})` + `rotate(90deg)`
```

Comment:
Typically, best practice is not to use random integers, such as 58, to make things "work" if they have no direct meaning.  That being said, this may be a valid solution to the issue. @Acumane if you could weigh in on this solution that'd be awesome.